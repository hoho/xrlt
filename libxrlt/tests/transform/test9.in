id:0, type:300, last:0, error: 0, data:AAABBB
id:0, type:300, last:0, error: 0, data:CCCDDD
id:0, type:200, last:0, error: 0, data:EEEFFF
id:0, type:200, last:0, error: 0, data:GGGHHH
id:0, type:400, last:0, error: 0, data:302
id:0, type:500, last:0, error: 0, data:hello world
id:0, type:600, last:1, error: 0, data:{"a": 123, "b": 456}
id:0, type:0, last:0, error: 0, data:
