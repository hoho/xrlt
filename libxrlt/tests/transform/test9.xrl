<?xml version="1.0"?>
<xrl:requestsheet xmlns:xrl="http://xrlt.net/Transform">

    <xrl:querystring name="qs" type="text" />
    <xrl:body name="b" type="json" />

    <xrl:response>
        <xrl:header name="EEE" />|<xrl:cookie name="CCC" />|<xrl:status />
        <xrl:text>|</xrl:text>
        <xrl:value-of select="$qs" />|<xrl:value-of select="$b" />
    </xrl:response>

</xrl:requestsheet>
