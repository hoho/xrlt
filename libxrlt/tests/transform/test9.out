XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_WAITING
XRLT_STATUS_CHUNK
chunk: FFF
chunk: |
chunk: DDD
chunk: |
chunk: hello world
chunk: |
chunk: 123456
XRLT_STATUS_DONE
